tests/test__collection_api.py

```diff
@@ -56,12 +56,10 @@ def test__get_collection_full_name(self):
     def test__get_collection_names(self):
```

Comment:
Please keep at least one test calling collection_names: even if it's deprecated, it's still implemented by pymongo.

Reply:
That should be fixed now. Thanks for the input